src/ACP10Ar/MotionControl/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@loupeteam/acp10motionprog",
+  "version": "0.1.0",
+  "description": "This is a program package that will add the ACP10 Servo Manager program to your AS project",
+  "author": "Loupe",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/loupeteam/AxisLib.git"
+  },
+  "dependencies": {
+    "@loupeteam/axiserrlib": "0.0.6",
+    "@loupeteam/errorprog": "^0.1.0",
+    "@loupeteam/firstinitprog": "^0.1.1",
+    "@loupeteam/piperprog": "^0.1.0",
+    "@loupeteam/axislib": "3.7.0"
```

Comment:
Have decided that we don't want to split axislib into two packages (e.g. `axislibacp10` and `axislibmapp`) and instead want to use the major version to handle it, acp10 always having versions 3.y.z and mapp having >= 4.y.z ?

I'm guessing Jenkins wouldn't have a problem processing two different versions of the same package in a single repo...